workshops/util.py

```diff
@@ -1,5 +1,11 @@
+import csv
 from math import pi, sin, cos, acos
 
+from .models import Event, Role, Person
+
+PERSON_UPLOAD_FIELDS = ['personal', 'middle', 'family', 'email']
+PERSON_TASK_UPLOAD_FIELDS = PERSON_UPLOAD_FIELDS + ['event', 'role']
```

Comment:
Please make `PERSON_UPLOAD_FIELDS` and `PERSON_TASK_UPLOAD_FIELDS` immutable by turning them into tuples.

Additionally you can move them into `Person` model definition as stated here: https://github.com/swcarpentry/amy/pull/165#discussion_r23534853